src/routes/image.rs

```diff
@@ -71,6 +71,8 @@ pub enum ImageProcessingError {
     ProcessingWorkerJoinError,
     #[error("the image processing with libvips has failed")]
     LibvipsProcessingFailed(libvips::error::Error),
+    #[error("The image exceeds the allowed size")]
+    FileSizeExceeded,
```

Comment:
Please add a `u32` parameter to this error so you could return in the error message the maximum allowed size as well:
```rust
FileSizeExceeded(u32)
```